src/Spawner.cpp

```diff
@@ -256,7 +233,41 @@ void Spawner::ProcessHandle::sendSigTerm() const
     }
 }
 
+bool Spawner::ProcessHandle::end()
```

Comment:
What's the reason for having this function as well as `killDeployment` + `wait()`. Looks like is a re-implementaion of the same thing (also re-implementing the `sendSig*` functions), and it looks like `end()` is called nowhere.. so I wonder what's its purpose.

Reply:
Completely true, I forgot to remove this one. It is now gone.